<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ome_maps_select_dialog</class>
 <widget class="QDialog" name="ome_maps_select_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>295</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0" colspan="2">
    <widget class="QTabWidget" name="tab_widget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="load_tab">
      <attribute name="title">
       <string>Load</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="2">
        <widget class="QPushButton" name="select_file_button">
         <property name="text">
          <string>Select File</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLineEdit" name="file_name"/>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QLabel" name="load_note">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;The selected material must exactly match the one used to generate this file&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="generate_tab">
      <attribute name="title">
       <string>Generate</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0">
        <widget class="QCheckBox" name="apply_threshold">
         <property name="text">
          <string>Threshold data?</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="ScientificDoubleSpinBox" name="threshold">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="decimals">
          <number>8</number>
         </property>
         <property name="maximum">
          <double>100000.000000000000000</double>
         </property>
         <property name="value">
          <double>250.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="num_hkls_selected">
         <property name="text">
          <string>Number of hkls selected:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="choose_hkls">
         <property name="text">
          <string>Choose HKLs</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="method_label">
     <property name="text">
      <string>Method:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="method">
     <item>
      <property name="text">
       <string>Load</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Generate</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="material_label">
     <property name="text">
      <string>Selected Material:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="material"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScientificDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>scientificspinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>material</tabstop>
  <tabstop>method</tabstop>
  <tabstop>tab_widget</tabstop>
  <tabstop>file_name</tabstop>
  <tabstop>select_file_button</tabstop>
  <tabstop>choose_hkls</tabstop>
  <tabstop>apply_threshold</tabstop>
  <tabstop>threshold</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>ome_maps_select_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>ome_maps_select_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>apply_threshold</sender>
   <signal>toggled(bool)</signal>
   <receiver>threshold</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
